build-and-test-with-yarn/action.yaml

```
@@ -24,13 +24,25 @@ inputs:
     description: Comma-separated paths to directories containing main source files.
     default: src/
     required: false
-  sonar-test-exclusions:
-    description: One or more path matching patterns defining files to be excluded from analysis, among those found below the directories specified in `sonar.tests`.
-    default: test/**
+  sonar-inclusions:
+    description: One or more path matching patterns defining files to be included in analysis, among those found below directories specified in `sonar.sources`.
+    default: "**/*.ts"
+    required: false
+  sonar-exclusions:
+    description: One or more path matching patterns defining files to be excluded from analysis, among those found below the directories specified in `sonar.sources`.
+    default: "**/*.spec.ts"
```

Dont we use schema test.ts?
```suggestion
    default: "**/*.test.ts"
```

---

see comment above.

---

As agreed during the Daily I have amended this to match `test.ts`